modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/internal/jaxrs/application/FDSApplication.java

```diff
@@ -179,6 +183,37 @@ public Response saveActiveFDSViewSettings(
 		).build();
 	}
 
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Path("/fds/{fdsName}/custom-views")
+	@POST
+	public Response saveFDSCustomView(
```

Comment:
Should we generate a REST API with the rest builder for this, or we want to keep in this way? I'm just asking, maybe it is not necessary to generate it with the restBuilder because these endpoints will be use only for intern calls.

Reply:
Yes! See https://github.com/liferay-frontend/liferay-portal/pull/2526#discussion_r921969552 

We can do this in a separate task.